### PR TITLE
New annotation patterns applied: seeAlso, isDefinedBy

### PR DESCRIPTION
**Pull request by mereolog**
This applies the new patterns from the two previous pull requests.

### CODE REVIEW

**Review by bszady — DISMISSED** [review]
For me seems ok. @przemekgradzki please make a quick review.

File: ontohgis.ttl [hunk 1]
```diff
@@ -10282,9 +10283,9 @@ ontohgis:object_23 rdf:type owl:Class ;
                    rdfs:subClassOf ontohgis:object_1 ;
                    rdfs:label "budynek"@pl ,
                               "building"@en ;
-                   rdfs:seeAlso <http://gov.genealogy.net/types.owl#17> ,
-                                "http://encyklopedia.pwn.pl/szukaj/budynek.html" ;
-                   ontohgis:isDefinedByLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl .
+                   ontohgis:isDefinedByLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
+                   ontohgis:seeAlsoIRI <http://gov.genealogy.net/types.owl#17> ;
+                    ontohgis:seeAlsoLiteral                   "http://encyklopedia.pwn.pl/szukaj/budynek.html" .
```
Comment (bernardyn) [inline ontohgis.ttl:10288]: it should be seeAlsoIRI in my opinion

**Review by bernardyn — COMMENTED** [review]
only one exception marked

File: ontohgis.ttl (hunk 1 above)
Comment (mereolog) [reply ontohgis.ttl:10288]: > it should be seeAlsoIRI in my opinion

Fixed now.

**Review by bernardyn — APPROVED** [review]
Approved

**Pull request merged**